sdk/template-dpg/Azure.ServiceTemplate.Template/content/test-resources.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I didn't see this file referenced in the getting started guide -- do we need to include it here?

Reply:
Same as above, for CI